src/components/search/QASearchResults.jsx

```diff
@@ -131,7 +120,22 @@ const QASearchResults = (props) => {
       }
       <div className="col-sm-2"></div>
       <div className="col-sm-8">
-        <BootstrapTable id="search-results-list" keyField="uri" data={ tableData } columns={ columns } />
+        <table className="table table-bordered">
+          <thead>
+            <tr>
+              <th className="sinopia" style={{ width: '45%' }}>
```

Comment:
Doesn't seem like a good choice for class name.

Reply:
In the main.css it is scoped to just the **th** element, happy to change the name